grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -120,6 +120,12 @@ class StockMovementController {
         [stockMovement: stockMovement]
     }
 
+    def createPicklist = {
```

Comment:
I think it should be named `transitionToPicking` or sth like that because you are not only creating the picklist here